tuf/data/root_test.go

```diff
@@ -52,7 +54,7 @@ func validRootTemplate() *SignedRoot {
 }
 
 func TestRootToSignedMarshalsSignedPortionWithCanonicalJSON(t *testing.T) {
-	r := SignedRoot{Signed: Root{Type: "root", Version: 2, Expires: time.Now()}}
+	r := SignedRoot{Signed: Root{SignedCommon: SignedCommon{Type: "root", Version: 2, Expires: time.Now()}}}
```

Comment:
nit: should this be uppercase `"Root"`?  It shouldn't affect the behavior of this test but it seems like we should keep this consistent with what is expected (there are a few other instances in this file)